### PR TITLE
Style sheet for the panel title bar

### PR DESCRIPTION
**Pull request by shun-iwasawa**
This PR is for the issue #696 
With this modification users can modify the panel's title bar by using style sheets.

- Specify 4 pixmaps for the border image of the title bar with (floating/docked) x (active/inactive) = 4 alternative states.
- Specify 2 colors for the text color of title bar with active/inactive states.

Note that I also changed the solid white line appeared on the edge of active title bar to light gray, reflecting @konero 's feedback.

### CODE REVIEW

**Comment by turtletooth** [comment]
I'm going to look at this tonight.

**Comment by turtletooth** [comment]
@shun-iwasawa What are your thoughts on renaming the current themes to something more descriptive like light, medium and dark?

**Comment by turtletooth** [comment]
If someone can call Jenkins on this, I've tested it here and it looks good.

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by shun-iwasawa** [comment]
> What are your thoughts on renaming the current themes to something more descriptive like light, medium and dark?

@turtletooth 
I have no objection to change the style name if it will become more clear for users. I named gray_### for each style because in Toonz Harlequin there already were "light" and "medium" styles.
Note that the "light" style of Toonz Harlequin is much whiter than 128-gray, it's 225-gray based. Even the "medium" style is 140-gray based. So one thing I am a little worried about is that someone may puzzle about a style name when he would like to introduce a brighter interface in the future...
Anyway, basically I agree with your suggestion.

**Comment by turtletooth** [comment]
> Note that the "light" style of Toonz Harlequin is much whiter than 128-gray, it's 225-gray based

I thought about that too and agree that the light, medium and dark aren't the most accurate names.  I will think about the better names.  I just think that something more descriptive for the user would be good.

**Comment by turtletooth** [comment]
LGTM.  Thanks @shun-iwasawa

**Comment by konero** [comment]
Thanks @shun-iwasawa !

Unfortunately when we looked at the latest nightly containing this PR it seems to be missing. I'll try again on Monday.

**Comment by turtletooth** [comment]
@konero I think it may be my fault. When I went to merge it, I accidentally hit the close button.  I reopened and merged it, but it may have messed something up in the automatic build for the most recent nightly.  The next nightly should have the pr.  I'm sorry for my mistake.
